dolfyn/io/nortek.py

```diff
@@ -74,9 +83,23 @@ def read_nortek(filename,
     rdr.dat2sci()
     dat = rdr.data
 
+    od = dat['orient']
+    if 'orient.orientmat' not in dat:
```

Comment:
Add comment like:  The binary files from Nortek Vectors with an IMU will output orient.orientmat. Binary files from Nortek Vectors without an IMU and Nortek ADCPs (??) will not have an orient.orientmat, but rather heading, pitch, and roll values. Here, we make an orientmat using heading, pitch, roll as inputs.